packages/core/src/index.ts

```diff
@@ -130,10 +135,13 @@ const buildVersionStr = ({
 };
 
 export const endAppStartup = (properties?: Properties): Promise<boolean> => {
```

Comment:
`endAppStartup` can be removed, see https://embrace-io.slack.com/archives/C07CSGLSARJ/p1721665753052099

Reply:
I slacked you, because I think we need to recreate it for ios